mission_yaml/SHW001_M34.yml

```diff
@@ -0,0 +1,330 @@
+metadata:
+  acknowledgement: This study used data collected and made freely available by Voice
+    of the Ocean Foundation (https://voiceoftheocean.org) accessed from https://erddap.observations.voiceoftheocean.org/erddap/index.html
+  institution: Voice of the Ocean Foundation
+  license: Creative Commons Attribution 4.0 (https://creativecommons.org/licenses/by/4.0/)
+    This study used data collected and made freely available by Voice of the Ocean
+    Foundation (https://voiceoftheocean.org) accessed from https://erddap.observations.voiceoftheocean.org/erddap/index.html
+  format_version: IOOS_Glider_NetCDF_v2.0.nc
+  glider_model: SeaExplorer
```

Comment:
I think glider_model should be SeaExplorer Shallow, (https://github.com/OceanGlidersCommunity/OG-format-user-manual/pull/277)
We'll probably revisit it later, but just to distinguish it from our regular gliders